joint1_controller:
    controller:
        package: dynamixel_controllers
        module: joint_torque_controller
        type: JointTorqueController
    joint_name: joint1
    joint_speed: 0.3
    motor:
        id: 4
        init: 2047
        min: 0
        max: 4095

